apps/backend/prisma/schema.prisma

```diff
@@ -13,7 +13,68 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum Priority {
+  LOW
+  MEDIUM
+  HIGH
+}
+
+enum Status {
+  OPEN
+  IN_PROGRESS
+  DONE
+}
+
+enum Role {
+  ADMIN
+  USER
+}
+
 model User {
-  authSchId String @id
-  email     String @unique
+  id             Int     @id @default(autoincrement())
+  authSchId      String //Ez majd a belepeshez kell
+  email          String  @unique
+  profilePicture String?
+  groups         GroupMembers[]
+  events         Event[]
+}
+
+model Group {
+  id          Int     @id @default(autoincrement())
+  name        String
+  description String?
+  members     GroupMembers[]
+  events      Event[]
+}
+
+model GroupMembers {
+  groupId Int
+  userId  Int
+  role    Role @default(USER)
+
+  @@id([groupId, userId])
+  Group   Group @relation(fields: [groupId], references: [id])
+  User    User  @relation(fields: [userId], references: [id])
+}
+
+model Event {
+  id           Int      @id @default(autoincrement())
+  name         String
+  description  String?
+  date         DateTime @default(now())
```

Comment:
Arról nem beszeltunk, hogy mikor akarunk idopontot is tarolni. Szerintem alapbol csak a napot adjuk meg, és opcionalisan a kezdo, valamint opcionalisan a vegpontot. Ehhez hasznalhato a prisma @db.Date es @db.Time
 [doksi](https://www.prisma.io/docs/orm/reference/prisma-schema-reference#datetime)

Reply:
Akkor egyelore erre felveszek 4 mezot (startDate, startTime, endDate, endTime), de ezt szerintem lehet optimalisabban majd a jovoben